tome/users/migrations/0019_participants_display_name.py

```
@@ -0,0 +1,36 @@
+# Participants display_name: add column, backfill from name, then set unique
+
+from django.db import migrations, models
+
+
+def backfill_display_name(apps, schema_editor):
```

Ruff (0.15.2)

[warning] Unused function argument: `schema_editor`

(ARG001)